schema/src/main/jade-fragments/hles_dog_residential_environment.fragment.json

```diff
@@ -0,0 +1,397 @@
+{
+  "name": "hles_dog_residential_environment",
+  "columns": [
+    {
+      "name": "de_lifetime_residence_count",
+      "datatype": "integer"
+    },
+    {
+      "name": "de_past_residence_zip_count",
+      "datatype": "integer"
+    },
+    {
+      "name": "de_past_residence_zip_1",
```

Comment:
Are these names going to run into the same round-trip problem w/ camel casing? Should we do `zip1` for now to keep working around it?